.github/workflows/ci.yml

```diff
@@ -14,21 +14,15 @@ concurrency:
 
 jobs:
   test:
-    name: test ${{ matrix.rust }} ${{ matrix.flags }}
-    runs-on: ubuntu-latest
+    name: test ${{ matrix.os}} ${{ matrix.rust }} ${{ matrix.flags }}
     timeout-minutes: 30
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        rust: ["stable", "beta", "nightly"]
+        os: ["ubuntu-latest", "windows-latest"]
+        rust: ["stable", "nightly", "1.76"] # MSRV
         flags: ["--no-default-features", "", "--all-features"]
-        include:
```

Comment:
This was `include`d because we don't want to run `--all-features` for MSRV, as we don't guarantee that feature flags have the same MSRV.